### DISCUSSION TITLE
How to query a nested collection

### DISCUSSION THREAD

**Discussion by BearDooks** [GodotNuts/GodotFirebase#167] [Q&A] [answered]
This is from @DanielVZ96 on Issue #97 

@WolfgangSenff I'm sorry but can you give some example on how you'd query nested collections?

**Comment by WolfgangSenff** [comment, marked as answer]
@DanielVZ96 - the key here is to have a reference down into the nested collection itself already. At the moment, if there is a nested collection and you have the *parent* document, you cannot get documents inside the nested collection. The team will be discussing how to handle that scenario, if we even want to, but I do not have a good bit of sample code to show you - just have to play around and make sure your path nests all the way to the correct document.
